cmd/rdeps/rdeps.go

```diff
@@ -22,6 +21,7 @@ func main() {
 	var tags buildutil.TagsFlag
 	flag.Var(&tags, "tags", "List of build tags")
 	stdin := flag.Bool("stdin", false, "Read packages from stdin instead of the command line")
+	indirect := flag.Bool("indirect", false, "Print also indirect dependencies")
```

Comment:
Call the flag `r` (for recursive), with the description `Print reverse dependencies recursively`. Call the variable `recursive`.